project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Template for presenting additional information fields.
+ *
+ * Type, Status and Access at content group header title.
+ */
+?>
+<div class="row" id="content-group-header-title">
+  <div class="col-sm-12 col-md-8">
+    <h1><?php print $title;?></h1>
+  </div>
+  <div class="col-sm-12 col-md-4" id="content-group-header-title-right-div">
+    <div class="row">
+      <div class="col-xs-offset-7 col-xs-5">
+        <div class="top-buffer indication indication-type">
+          group
```

Comment:
OK for now. Projects have different theming, but might re-use the title. If so, we'll refactor here.